vite.config.ts

```diff
@@ -37,17 +34,17 @@ export default defineConfig({
         WindiCSS()
     ],
     server: {
-        port: parseInt(process.env.APP_PORT),
+        port: parseInt(process.env.SERVER_PORT),
         strictPort: true,
-        host: process.env.APP_HOST,
+        host: process.env.SERVER_HOST,
     },
     build: {
         rollupOptions: {
             plugins: [
                 gzipPlugin(),
-                // Uncomment to view bundles when running `npm run build`
-                // analyze({summaryOnly: true}),
-            ]
+            ].concat(process.env.NODE_ENV === 'production' ? [
```

Comment:
I would prefer `import.meta.env.PROD` as per https://vitejs.dev/guide/env-and-mode